src/tags/object/RichText/model.js

```diff
@@ -218,31 +268,85 @@ const Model = types
 
         self.setReady(false);
 
-        // init and render regions into working node, then move them to visible one
-        beforeNeedsUpdateCallback?.();
-        self.regs.forEach(region => {
-          try {
-            // will be initialized only once
-            region.initRangeAndOffsets();
-            region.applyHighlight();
-          } catch (err) {
-            console.error(err);
-          }
-        });
-        afterNeedsUpdateCallback?.();
-
-        // node texts can be only retrieved from the visible node
-        self.regs.forEach(region => {
-          try {
-            region.updateHighlightedText();
-          } catch (err) {
-            console.error(err);
-          }
-        });
+        if (!isFF(FF_LSDV_4620_3)) {
```

Comment:
Maybe switch the if/else around so it reads in the affirmative case (have `if (isFF(FF_LSDV_4620_3)) {` come first), especially given this new behaviour we are hoping to make the official and only one, and it should be read first anywhere we are looking at this code.